### DISCUSSION TITLE
How to pass a prefix for the persist store key

### DISCUSSION THREAD

**Discussion by emadabdulrahim** [pmndrs/zustand#2009] [Q&A] [answered]
I want to name space the local storage key based on `orgId` which I can only know at run time. What's the correct way to pass that as an argument to zustand's `create` function? I'm on v4 and using `create`. I know how to initialize store with `createStore`, but not with `create`.

Or perhaps I have to use `createStore` and use Context in this case? 🤔

Thanks

**Comment by dbritto-dev** [comment, marked as answer]
The only way is using `createStore()`
